library/src/main/java/com/schibsted/spain/barista/assertion/BaristaVisibilityAssertions.kt

```diff
@@ -70,4 +74,28 @@ object BaristaVisibilityAssertions{
         onView(withText(text)).check(matches(not(isDisplayed())))
     }
 
+    @JvmStatic
+    fun assertContains(text: String) {
+        assertDisplayed(withText(CoreMatchers.containsString(text)))
+    }
+
+    @JvmStatic
+    fun assertContains(@IdRes resId: Int, text: String) {
+        onView(withId(resId)).check(matches(withText(containsString(text))))
+    }
+
+    @JvmStatic
+    fun assertNotContains(text: String) {
+        onView(withText(containsString(text))).check(matches(not(isDisplayed())))
+    }
+
+    @JvmStatic
+    fun assertTextColor(@IdRes resId: Int, @ColorRes colorRes: Int) {
+        onView(withId(resId)).check(matches(TextColorMatcher(colorRes)))
+    }
+
+    @JvmStatic
+    fun assertNotTextColor(@IdRes resId: Int, @ColorRes colorRes: Int) {
```

Comment:
Something deep inside me tells me that `assertNotTextColor` is not very english-friendly. But I'm not sure how it should be 🤔

Reply:
assertTextColorIs/assertTextColorIsNot or assertTextColorEquals/assertTextColorNotEqual?

Reply:
Like it!!